hail/src/main/scala/is/hail/io/vcf/ExportVCF.scala

```diff
@@ -453,5 +460,52 @@ object ExportVCF {
         sb.result()
       }
     }.writeTable(ctx, path, Some(header), exportType = exportType)
+
+    if (tabix) {
+      exportType match {
+        case ExportType.CONCATENATED =>
+          info(s"Writing tabix index for $path")
+          TabixVCF(ctx.fsBc, path)
+        case ExportType.PARALLEL_SEPARATE_HEADER | ExportType.PARALLEL_HEADER_IN_SHARD =>
+          val localFsBc = ctx.fsBc
+          val files = fs.glob(path + "/part-*").map(_.getPath.getBytes)
+          info(s"Writing tabix index for $path")
+          ctx.backend.parallelizeAndComputeWithIndex(ctx.backendContext, files) { (pathBytes, _) =>
+            TabixVCF(localFsBc, new String(pathBytes))
+            null
```

Comment:
how does this work? A quick read of the parallelizeAndComputeWithIndex indicates this may be problematic

Reply:
Maybe Array.empty[Byte] instead?

Reply:
Yeah, nonnull would probably be better here.